dart/test/http_client/sentry_http_client_test.dart

```diff
@@ -199,7 +176,45 @@ void main() {
       expect(mockHub.captureExceptionCalls.length, 0);
       verify(mockClient.close());
     });
+
+    test('Breadcrumb has correct duration', () async {
+      final sut = fixture.getSut(MockClient((request) async {
+        expect(request.url, requestUri);
+        await Future.delayed(Duration(seconds: 1));
+        return Response('', 200, reasonPhrase: 'OK');
+      }));
+
+      final response = await sut.get(requestUri);
+      expect(response.statusCode, 200);
+
+      expect(fixture.hub.addBreadcrumbCalls.length, 1);
+      final breadcrumb = fixture.hub.addBreadcrumbCalls.first.crumb;
+
+      var durationString = breadcrumb.data!['duration']! as String;
+      // we don't check for anything below a second
+      expect(durationString.startsWith('0:00:01'), true);
+    });
   });
 }
 
 class CloseableMockClient extends Mock implements BaseClient {}
+
+class Fixture {
+  // every test does final sut = fixture.getSut() either passing MockClient or not
+  SentryHttpClient getSut([MockClient? client]) {
+    final mc = client ?? getClient();
+    return SentryHttpClient(client: mc, hub: hub);
+  }
+
+  // tests could assert or expect fixture.hub
+  late MockHub hub = MockHub();
+
+  // could optionally have reasonPhrase, status etc with defaults values
+  // so tests could also do sut = fixture.getSut(fixture.getClient(500, 'my reason'))
```

Comment:
the goal of the comments were just to explain you, we can remove from source code